include/Track.h

```diff
@@ -133,6 +133,43 @@ class LMMS_EXPORT TrackContentObject : public Model, public JournallingObject
 	{
 		return m_autoResize;
 	}
+	
+	unsigned int color() const
+	{
+		return( m_color.rgb() );
+	}
+	
+	QColor colorObj() const
+	{
+		return m_color;
+	}
+
+	void setColor( const QColor & c )
+	{
+		m_color = QColor( c );
+	}
+	
+	void setColorRgb( const unsigned int & c )
+	{
+		m_color.setRgb( c );
+	}
+
+	void setUseStyleColor( bool b )
+	{
+		m_useStyleColor = b;
+	}
+	
+	bool useStyleColor()
+	{
+		return m_useStyleColor;
+	}
+	
+	bool useCustomClipColor()
+	{
+		return m_useCustomClipColor;
+	}
+	
+	void setUseCustomClipColor( bool b );
```

Comment:
```suggestion
	void useStyleColor( bool b )
	{
		m_useStyleColor = b;
	}
	
	bool usesStyleColor()
	{
		return m_useStyleColor;
	}
	
	void useCustomClipColor( bool b );
	
	bool usesCustomClipColor()
	{
		return m_useCustomClipColor;
	}

```

I would prefer this, or just "set/get" prefixes on everything, `use__` doesn't sound like a getter to me.

Reply:
IMO setX and X are more compliant with the rest of the code, but I suppose I could change these if you wish.

Reply:
Dropping the "get" from a getter works a lot better when the the variable name is clearly a property of some sort (ex. children() / height() / active()).

useStyleColor() sounds like it would set m_useStyleColor to true, or take a boolean argument and set m_useStyleColor to it.

If you were to rename the variable to m_hasCustomColor, you suddenly get a more intuitive getter name, hasCustomColor(). I think this might be the best solution.

(A quick categorization of my opinion on getter names (that hopefully doesn't have huge flaws):
- Good: Nouns, adjectives, third person verbs
- Bad: First + second person verbs
)

Reply:
I changed it to the suggested naming scheme, but renaming to m_hasCustomColor is something I shall look over (m_useCustomClipColor may be confused with this, and then there's all the logic I'll have to rewrite).